fiftyone/operators/builtins/panels/model_evaluation/__init__.py

```
@@ -134,6 +134,12 @@ def get_map(self, results):
         except Exception as e:
             return None
 
+    def get_mar(self, results):
+        try:
+            return results.mAR()
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)